SonarQube.TeamBuild.PostProcessor/CoverageReportProcessor.cs

```diff
@@ -28,33 +37,43 @@ public bool ProcessCoverageReports(AnalysisConfig config, TeamBuildSettings sett
                 throw new ArgumentNullException("logger");
```

Comment:
Existing error: the config value is being checked but the error message says "logger". This class doesn't actually use either of those directly; the only one it cares about is settings, so it should check that.